File: benchmark/corpus/03_div_zero/b_bucket_remainder.py
# Buckets an id with a remainder; the bucket size comes from input.
import sys


def bucket(id, size):
    return id % size  # DANGER


line = sys.stdin.readline().strip()
try:
    size = int(line)
except ValueError:
    print("not a number: " + line)
else:
    print("bucket %d" % bucket(1234, size))

File: benchmark/corpus/03_div_zero/b_bucket_remainder.js
// Buckets an id with a remainder; the bucket size comes from input.
function bucket(id, size) {
  return id % size; // DANGER
}

const line = (await new Response(Deno.stdin.readable).text()).trim();
const size = Number(line);
if (Number.isNaN(size)) console.log("not a number: " + line);
else console.log("bucket " + bucket(1234, size));
